docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eks-helm.mdx

```diff
@@ -0,0 +1,360 @@
+---
+id: eks-helm
+title: "EKS Helm Setup"
+description: "Guide on setting up required resources with Helm"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Prerequisites
+
+- a Kubernetes Cluster, see [eksctl](#) or [terraform](#) guide <!-- TODO: replace with proper links -->
+- [Helm (3.13+)](https://helm.sh/docs/intro/install/) installed
+- [kubectl (1.28+)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
+- (optional) Domain name / [hosted zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-working-with.html) in Route53
+  - this allows to expose Camunda 8 and easily connect via [zbctl](../../../../../../apis-tools/cli-client/) or the [Camunda Modeler](https://camunda.com/download/modeler/)
+
+## Considerations
+
+### Architecture
+
+![Camunda 8 Self-Managed AWS Architecture Diagram](./assets/camunda-8-self-managed-architecture-aws.png)
+
+## Usage
+
+### Environment Prerequisites
+
+In order to streamline the execution of the subsequent commands, it is recommended to export multiple environment variables.
+
+The following are the required environment variables with some example values.
+
+```shell
+# Your standard region that you host AWS resources in
+export REGION=eu-central-1
+# Following two environment variables can be skipped if you don't have a domain
+# The domain name that you intend to use
+export DOMAIN_NAME=camunda.example.com
+# The e-mail to register with Let's Encrypt
+export MAIL=admin@camunda.example.com
+# The Ingress Helm Chart version
+export INGRESS_HELM_CHART_VERSION=4.8.3
+# The External DNS Helm Chart version
+export EXTERNAL_DNS_HELM_CHART_VERSION=1.13.1
+# The Cert-Manager Helm Chart version
+export CERT_MANAGER_HELM_CHART_VERSION=1.13.2
+# The Camunda 8 Helm Chart version
+export CAMUNDA_HELM_CHART_VERSION=8.3.3
+```
+
+Additionally, please follow the guide from either [eksctl](#) or from [Terraform](#) to retrieve the following values, which will be required for subsequent steps. <!-- TODO: Link to sources -->
+
+- EXTERNAL_DNS_IRSA_ARN
+- CERT_MANAGER_IRSA_ARN
+- DB_HOST
+- PG_USERNAME
+- PG_PASSWORD
+- DEFAULT_DB_NAME
+- REGION
+
+### DNS Setup
+
+:::info
+If you don't have a domain name, you can skip to [Camunda 8](#camunda-8).
+:::
+
+#### ingress-nginx
+
+Ingress-nginx is an open-source Kubernetes Ingress controller that provides a way to manage external access to services within a Kubernetes cluster. It acts as a reverse proxy and load balancer, routing incoming traffic to the appropriate services based on rules defined in the Ingress resource.
```

Comment:
let's add a link to the project so people are 100% aligned on which one we use: https://github.com/kubernetes/ingress-nginx